app/queries/validator_query.rb

```diff
@@ -21,7 +33,7 @@ def call(network: "mainnet", sort_order: "score", limit: 9999, page: 1, random_s
     scope = set_ordering(scope, sort_order, random_seed_val)
     scope = set_pagination(scope, page, limit)
 
-    @api ? scope : scope.scorable
+    query_params[:active_only] ? scope.scorable : scope
```

Comment:
This condition is incorrect. All strings, even those with "false" values, are truthy, so this condition will always return active validators.
Please add this scenario to tests - only the opposite scanario is tested.